commands.go

```diff
@@ -32,27 +34,27 @@ Read more about this bot and report bugs here: https://github.com/matvey00z/tele
 	)
 }
 
-func getStatsString(total int64, reactionsCnt [len(reactions)]int64) string {
+func getStatsString(total int64, reactionsCnt [len(reactions)]int64, ReactionsPercent [len(reactions)]float64) string {
```

Comment:
1. Please follow naming conventions
2. `ReactionsPercent` is passed to the function but not used